### ISSUE TITLE
New version support past 1.19.2, and old version support prior to 1.19

### ISSUE THREAD

**Issue by GoldenFurredFox** [TeamAOF/DarkModeEverywhere#5]
Gonna say right now, this is not a duplicate of #2 since I'm asking for 1.19.3 through 1.20.1 as well as 1.18.1 and 1.18.2.

Please add support for the versions that the Forge version supports.

**Comment by Kazuhiko-Gushiken** [comment]
Version 1.19.3 and 1.19.4 changed a lot of things and are not big versions. A lot of mods have not updated beyond 1.19.2 within the 1.19 versions so if they did update, there would be a mass amount of mods missing.

As for 1.20, we are currently waiting on most of the mods to stably update to 1.20 version, but we are already planning All of Fabric 7 for 1.20; though it will take time as we're dependent on other mods updating.
